_posts/2024-06-30-simplifying-services.markdown

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title:  "Simplifying a key-value service using CHERIoT"
+date:   2024-06-30
+categories: ISA Ibex
```

Comment:
I'm not good at these, but maybe 'development rtos' would be a better fit here.  I don't think there's anything about Ibex and not much about the ISA.

```suggestion
categories: development rtos
```

Reply:
Done